lib/rack/session/constants.rb

```diff
@@ -1,6 +1,7 @@
-
 module Rack
   module Session
+    RACK_SESSION                        = 'rack.session'
```

Comment:
These keys are used by `Rack::Lint` so they remain in `rack/constants` IIRC. So I'm not sure we need to define them here a 2nd time? WDYT?

Reply:
It depends.  If the goal is to eventually make rack-session independent of rack, we probably want to do that.  If the idea is that rack-session will always depend on rack, then maybe there isn't a need.

Reply:
`RACK_SESSION` is for `Rack::Lint`. Anyone can write according to the spec for session management, `rack-session` is just one implementation.

Reply:
I didn't say we should remove the constant from rack.  As I wrote, if the goal is to eventually make rack-session not depend on rack, we will need to have the constant in rack-session.  Can you let me know whether making rack-session not depend on rack is a goal?

Reply:
I don’t have a strong opinion about it. Is there any reason why it should depend on rack? If not I guess it can be independent?

Reply:
I think a good future goal would be to make it not depend on rack, and having these constants works towards that.

Reply:
Okay, it makes sense.